api/envoy/config/trace/v3/skywalking.proto

```diff
@@ -0,0 +1,55 @@
+syntax = "proto3";
+
+package envoy.config.trace.v3;
+
+import "envoy/config/core/v3/grpc_service.proto";
+
+import "udpa/annotations/migrate.proto";
+import "udpa/annotations/status.proto";
+import "udpa/annotations/versioning.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.config.trace.v3";
+option java_outer_classname = "SkywalkingProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_migrate).move_to_package =
+    "envoy.extensions.tracers.skywalking.v4alpha";
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: SkyWalking tracer]
+
+// Configuration for the SkyWalking tracer.
+// [#extension: envoy.tracers.skywalking]
+message SkyWalkingConfig {
+  // SkyWalking collector service.
+  core.v3.GrpcService grpc_service = 1 [(validate.rules).message = {required: true}];
+
+  ClientConfig client_config = 2;
+}
+
+// Client config for SkyWalking tracer.
+// [#next-free-field: 6]
+message ClientConfig {
+  // Service name for SkyWalking tracer. If service_name is empty, then cluster
+  // name of Envoy will be used as service name.
+  string service_name = 1;
+
+  // Service instance name for SkyWalking tracer. If instance_name is empty, the
+  // node name of Envoy will be used as service instance name.
+  string instance_name = 2;
+
+  // Authentication token. Set client token if backend open token
+  // authentication.
+  string authentication = 3;
+
+  // When this field is set to true and Envoy is not the first tracing node, it
+  // directly uses the downstream endpoint as its own endpoint. This is helpful
+  // when the user wants to use the endpoint to aggregate all span data on the
+  // tracing link.
+  bool pass_endpoint = 4;
+
+  // Envoy caches the segment in memory when the backend service is temporarily
+  // unavailable. This field specifies the maximum number of segments that can
+  // be cached.
+  uint32 max_cache_size = 5;
```

Comment:
what's the default?

Reply:
The default value is 1024. I will add some new comment.